contracts/SpokePool.sol

```diff
@@ -1115,15 +1218,15 @@ abstract contract SpokePool is
 
     function _updateCountFromFill(
         uint256 startingFillAmount,
-        uint256 endingFillAmount,
+        bool localRepayment,
```

Comment:
so a first partial fill that is not a local repayment (i.e. with repayment on another chain) should NOT increment the fill counter, right? This is a good catch, seems really important actually and would have been bad if we missed it. Did you add a test for it?

Reply:
as discussed offline, i've merged the above comment